src/components/Register/Register.tsx

```diff
@@ -20,10 +22,13 @@ export const Register = (): JSX.Element => {
         validationSchema={validateRegisterSchema}
         onSubmit={(values, { resetForm }) => {
           resetForm({ values: initialValues });
```

Comment:
you should only reset the form in case the user was created successfully. If there is any error you can reset specific fields, but don't make the user fill the whole form again, it's a bad UX

Reply:
Will work on it, thank you